src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Data/SqlServer/ApplicationDbContextModelSnapshot.cs

```diff
@@ -16,7 +15,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "2.1.0-preview1-28193")
+                .HasAnnotation("ProductVersion", "2.1.0")
```

Comment:
Should this be updated to match what's building on dev?

Reply:
Sure I was going to do it in a PR after merging, but I can do this as part of the merge too, i'll update this PR